composer.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Unfortunately yes, if we intend to make it available as a [Composer] (https://getcomposer.org/) package via [Packagist] (https://packagist.org/) - de facto standard dependency manager in PHP world.